pkg/actuators/machine/reconciler.go

```diff
@@ -3,26 +3,26 @@ package machine
 import (
 	"fmt"
 	"strings"
+	"time"
 
 	nutanixClientV3 "github.com/nutanix-cloud-native/prism-go-client/pkg/nutanix/v3"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog/v2"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	configv1 "github.com/openshift/api/config/v1"
 	machinecontroller "github.com/openshift/machine-api-operator/pkg/controller/machine"
 	"github.com/openshift/machine-api-operator/pkg/metrics"
-	nutanixv1 "github.com/openshift/machine-api-provider-nutanix/pkg/apis/nutanixprovider/v1beta1"
 )
 
 const (
 	requeueAfterSeconds      = 20
 	requeueAfterFatalSeconds = 180
 	masterLabel              = "node-role.kubernetes.io/master"
 
-	providerIDFmt = "nutanix://%s"
-	infraName     = "cluster"
+	providerIDFormat = "nutanix://%s"
+
+	// MachineInstancePowerStateAnnotationName as annotation name for a machine instance power state
+	MachineInstancePowerStateAnnotationName = "machine.openshift.io/instance-power-state"
```

Comment:
it looks like this annotation is being used for creating logging, will it be used for something else as well?